VOCABULARY-CREATION-GUIDE.md

```diff
@@ -2,4 +2,33 @@
 
 Thing Description Forms can be extended with annotations on how to activate affordances.
```

Comment:
> activate affordances
gives the impression of only actuation on WoT devices, but with the property affordance we can also have sensing. Maybe we change this word to something which covers both cases. Suggestion from my side: `Thing Description Forms can be extended with annotations that further describe how to access affordances.`

Reply:
```suggestion
Thing Description Forms can be extended with annotations that further describe how to access affordances.
```